include/prism/node.h

```diff
@@ -112,6 +118,7 @@ PRISM_EXPORTED_FUNCTION const char * pm_node_type_to_str(pm_node_type_t node_typ
  * @param node The root node to start visiting from.
  * @param visitor The callback to call for each node in the subtree.
  * @param data An opaque pointer that is passed to the visitor callback.
+ * \public \memberof pm_node
```

Comment:
This one I disagree with, as it's more related to the visitor (the function callback). So I would exclude this.

Reply:
What do you think about switching it to "related"?

It'll still help discoverability, since someone looking at this API would probably look in `pm_node` to start